### ISSUE TITLE
Public Data Allows In-Game Cyberstalking/Bullying

### ISSUE THREAD

**Issue by Forman9** [jasonrohrer/OneLifeData7#1234]
In keeping with this quote from jason in the trust update:

"This doesn't allow you to track specific players across lives, but rather shows you the growing group of players that you generally trust."

Unfortunately, the public data available including curse/leaderboard names (as well as phex) in fact DOES allow people to track specific players across lives. This has led to many players on the discord, as well as phex groups, to track and harass players across the entire game space, as well as encourage others to curse them in an attempt to perma force them into donkeytown.

This is demonstrated clearly on this site [url]https://oholcurse.com/[/url]

The leaderboard tag, in combination with family information, allows a person to track where other players are. By combining the curse AND leaderboard tags, a player can track down a cursed player directly to their current life (where they will see the curse name above their head).

In the aforementioned trust update, Jason goes on to state...

"The curse system is great, insofar as it allows you to say, "I don't want to play near this person anymore," and have your wishes respected. Each person gets to decide what counts as "unacceptable" on their own, without needed consensus from others. Maybe one person doesn't like swearing, and another person doesn't like berry munching, and a third person really hates bossy players. Some people might like the drama of a murder or theft, while others might hate it. Curses allow us to avoid defining global behavior rules that must satisfy everyone."

I will repeat, " avoid defining global behavior rules that must satisfy everyone "

This is, of course, not at ALL what is happening, instead we have a clique of older, abusive players exploiting the various holes in the curse system (and public data) to try and enforce THEIR set of rules, which are really tantamount to "we don't like you".

They are then running around, tracking players down like a mob of thugs, harassing and mass cursing them, and encouraging others to do likewise. 

What's more, most of player base is probably TOTALLY UNWARE this is happening, just as I was unaware prior to joining the discord -- and that was after multiple years of playing the game.

I think it's fair to assume that, if prospective customers KNEW about this cyberstalking/bullying behavior (and the mechanics that enable it) PRIOR to buying the game, they wouldn't even bother.

**Comment by NotSoShady80** [comment]
Yeah I personally don't enjoy this and it negates the purpose of cursing, you want to get away from the person and not to ban them. This was the abuse I mentioned but never got wrote back on, I got multiple people talking to me about people going into group chats to stalk them and curse them randomly by tracking leaderboard data.

**Comment by DopiePanda** [comment]
Well if you are trying to hunt down someone you have cursed, then why even curse them in the first place? People curse to not have to be around these people anymore. And all cursing in game allows you to "track players between life", since you will see the same curse name every time.

Its funny seeing Shady crying about the community not liking him and standing up to his shitty behaviour, when he within the last 24 hours threatened to release someone elses private messages. 

But the kid isnt too sharp, at least if he does decide to commit a federal offence by releasing other peoples private communications, we won't have to worry about him here anymore as he will be busy in juvenile jail.

**Comment by NotSoShady80** [comment]
Dopie what the fuck are you on about with these private communications shit and all? This isn't standing up, I could leave anytime and the issue remains the same. Dopie you're listening to too many rumors

**Comment by DopiePanda** [comment]
What rumors? You are the one saying this stuff..

![image](https://github.com/jasonrohrer/OneLifeData7/assets/494864/f40c6b20-bbed-423d-b0fe-d8dc99fc6179)

**Comment by NotSoShady80** [comment]
the fuck are you on about, that's all public information you dumbass

**Comment by Rukhmar** [comment]
That is because you release boars on towns, Shady. We don't want our towns to be attacked so we warn others if we see you are around. If you didn't want to have people warning each other about you, don't do things that would make them want to warn others.

**Comment by NotSoShady80** [comment]
Do I release bores in the github?

**Comment by NotSoShady80** [comment]
Blackmail folder? lmfao, blackmail is blackmail, information is information. I'm not withdrawing nothing for payment, it's all in public and discord has all the information. Lol try the police and see what happens, half the ohol discord server is going to jail for child endangerment. You'd be telling on yourself, keep in mind all this information is public.

**Comment by GnihtonSeodYdahs** [comment]
Now it looks like you're all having schizophrenic conversation.

**Comment by GnihtonSeodYdahs** [comment]
I agree with this argument though me being Ydahs G.S.Y
